### PR TITLE
Fixed error in editing RSVP response

### LINKED ISSUE #311: Updating the RSVP of a meetup second time throws an error.
Steps to reproduce this
RSVP to a meetup. 
RSVP to the same meetup again.
Error Thrown 
image

### PR DESCRIPTION
**Pull request by sammy1997**
### Description
Whenever a user tries to edit his/her RSVP response after the 1st time, it threw an error. The error is now fixed.

Fixes #311

### Type of Change:
- Code
- Bug fix (non-breaking change which fixes an issue)

### How Has This Been Tested?
meetup 3
meetup 2
solved 1

### Checklist:
- [ ] My PR follows the style guidelines of this project
- [ ] I have performed a self-review of my own code or materials
- [ ] My changes generate no new warnings 
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by sammy1997** [comment]
@sa7i @tapasweni-pathak Can you please review this PR ?

File: systers_portal/meetup/views.py [hunk 1]
```diff
@@ -1004,6 +1004,9 @@ def get_form_kwargs(self):
         self.meetup = get_object_or_404(Meetup, slug=self.kwargs['meetup_slug'])
         kwargs.update({'user': self.request.user})
         kwargs.update({'meetup': self.meetup})
+        qs = Rsvp.objects.filter(user__user=self.request.user, meetup=self.meetup)
```
Comment (ajatprabha) [inline systers_portal/meetup/views.py]: Consider using a more descriptive variable name here.

**Comment by yatna** [comment]
Looks good, however as ajat mentioned try to give variables a better name to improve code readibility,

**Comment by sammy1997** [comment]
Sure @yatna I will change the name, and also add some tests. Thanks @yatna @ajatprabha

**Comment by sammy1997** [comment]
@abhi20161997 Can you review this once? I can't as it's my PR

**Comment by abhi20161997** [comment]
@sammy1997  Please can you use a more descriptive variable name for 'qs'. rest looks fine.

**Comment by sammy1997** [comment]
I am not getting time to work on this. I am closing this PR and making it available for the current students.

**Pull request closed without merging**